### DISCUSSION TITLE
/weekly

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#1943] [Nouvelles idées]
Pourquoi ne pas créer une commande /weekly qui, comme son nom l'indique, donne des récompenses une fois par semaine et je pensais faire en sorte que plus on vote au cours de la semaine, plus grande sera la récompense lors de la commande (je crois avoir vu passer qu'il fallait trouver une nouvelle utilité aux votes). 
On peut même faire que lorsque la commande est effectué au milieu de la semaine, le bot renvoi le délai restant avant de pouvoir récupérer les récompenses (ou simplement le jour) et la progression genre "8/14".

**Comment by QQtin3** [comment]
J'aime bien l'idée, on pourrait le faire n'importe quel jour de la semaine mais ça mets juste un CD de 7 days, pas besoin de forcément récupérer ça le Lundi par exemple.
À voir quelles récompenses mettre, comment les améliorer etc etc, mais je trouve l'idée sympa

**Comment by BastLast** [comment]
On risque de se séparer assez vite de top. Gg et donc de la mecanique de vote donc on risque pas d'investir du temps de dev dessus

**Reply by QQtin3** [reply]
Et autre que par les votes ? C'est pas parce que les votes vont plus fonctionner qu'on est obligé de baser cette feature sur les votes

**Reply by BastLast** [reply]
Ya deja une autrz discussion pr l'ajout de progressivité dans les récompenses journalières que javais mis en refuse je crois et je suis toujours pas chaud pr les meme raisons

**Reply by QQtin3** [reply]
Si c'est possible de tag la discussion à laquelle tu fais référence ça serait nice

**Reply by KirIcare** [reply]
La #1788 ?

**Reply by romain22222** [reply]
sinon, dans l'idée de récompenser le joueur vis à vis d'une action extérieure qui nous est bénéfique, je propose un système qui, si on est actif sur un des bots alpha / bêta, on obtient un truc cosmétique qu'on garde si on atteint les prérequis chaque semaine

Genre dans les affichages en ligne, on aurait un émoji star / sparkles / shooting star etc. qui apparaitrait à la fin de la ligne, signifiant que la personne est active sur les bots

**Reply by romain22222** [reply]
1- ca permettrait d'augmenter le nombre de tests sur la durée
2- ca inciterait les joueurs à venir sur le serv afin de tester les fonctionnalités
3- c'est gratifiant la reconnaissance

**Reply by BastLast** [reply]
Nan je visais plus sur un truc qui récompense si tu joue sur le serveur du bot
